webapp/constants/notifications.js

```diff
@@ -0,0 +1 @@
+export const NOTIFICATIONS_POLL_INTERVAL = 60000
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-20T21:02:01Z" title="Friday, September 20th 2019, 11:02:01 pm +02:00">Sep 20, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1 @@
+export const REFRECH_MILISEC = '60000'
```


```suggestion
export const REFRESH_MILLISEC = '60000'
```

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-09-21T05:52:20Z" title="Saturday, September 21st 2019, 7:52:20 am +02:00">Sep 21, 2019</time>_

---


Thanks for your hint @mattwr18 👍🏼